crux-llvm/src/Crux/LLVM/Simulate.hs

```diff
@@ -192,12 +215,12 @@ checkFun nm mp =
 ---------------------------------------------------------------------
 -- Profiling
 
-llvmMetrics :: forall arch p sym ext
-             . LLVMContext arch
+memMetrics :: forall p sym ext
+             . GlobalVar Mem
             -> Map.Map Text (Metric p sym ext)
-llvmMetrics llvmCtxt = Map.fromList [ ("LLVM.allocs", allocs)
-                                    , ("LLVM.writes", writes)
-                                    ]
+memMetrics memVar = Map.fromList [ ("LLVM.Mem.allocs", allocs)
+                                 , ("LLVM.Mem.writes", writes)
+                                 ]
```

Comment:
I'm curious if anything depended on these. I can't think of anything, but I have some vague memory that we had hooked up a JS profiler frontend to this at some point. I'm not sure if there was any special dependency of this data or other data.

Reply:
Ah, it was this https://github.com/GaloisInc/sympro-ui

Reply:
And yes, those fields are referenced by name https://github.com/GaloisInc/sympro-ui/blob/master/ts/profile.js#L12

Reply:
I did not know about sympro-ui, but these name changes were superfluous, so I've reverted them and added a note about the dependency.